package.json

```diff
@@ -17,19 +17,22 @@
     "testEnvironment": "./test-environment.js"
   },
   "scripts": {
-    "clean": "rm -Rf node_modules ./modules/*/node_modules ./modules/*/dist",
+    "clean": "rm -Rf node_modules ./modules/*/node_modules ./modules/*/dist ./modules/*/.turbo",
     "lint": "tsdx lint modules/*/src",
     "build": "yarn turbo run build",
-    "test": "TESTING=true tsdx test --verbose --runInBand",
+    "test": "yarn workspaces run test",
     "docs:client": "jsdoc2md --files ./modules/client/src/*.ts ./modules/client/src/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/client.md",
     "docs:addressList": "jsdoc2md --files ./modules/client/src/addressList/*.ts ./modules/client/src/addressList/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/addressList.md",
     "docs:erc20": "jsdoc2md --files ./modules/client/src/erc20/*.ts ./modules/client/src/erc20/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/erc20.md"
   },
   "devDependencies": {
     "@babel/preset-typescript": "^7.18.6",
+    "@ensdomains/ens-contracts": "^0.0.15",
+    "@types/node": "^18.11.9",
+    "ethers": "^5.7.2",
```

Comment:
I suggest we uninstall this and instead use the scoped packages only. Otherwise, we are bloating (yet more) the node_modules dependency hell

Reply:
1. this is in the root and not a module
2. we need ethers for the tests

Reply:
Can't the tests also use the `@ethersproject/xxx` subpackages as well?

Reply:
we need it because we run ganache and need to deploy locally all the contracts
that is nodejs stuff

Reply:
Please. We are importing the whole `ethers` package because one test uses `ethers.utils.defaultAbiCoder`

Reply:
removed. Just that you are aware of it. `@aragon/core-contracts-ethers` also imports ethers 5.7.2 as a complete package